src/Resources/config/services/form.xml

```diff
@@ -27,6 +27,10 @@
         <parameter key="setono_sylius_gift_card.form.type.add_gift_card_to_order.validation_groups" type="collection">
             <parameter>setono_sylius_gift_card</parameter>
         </parameter>
+
+        <parameter key="setono_sylius_gift_card.form.type.add_to_cart_gift_card_information.validation_groups" type="collection">
+            <parameter>sylius</parameter>
```

Comment:
```suggestion
            <parameter>setono_sylius_gift_card</parameter>
```

Reply:
It has to stay `sylius` because that is the group used to add product to cart. And we will need to validate that.

Second option would be to change the original validation groups. WDYT ?

Reply:
Stay with `sylius`, yes :)